File: 1.FDP/3° Competencia/C++/Repeticion/Ejercicio2.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el area de un rectangulo.\n";
	double Altura, Base, Area;
	int respuesta;
	cout << "Cuantas veces deceas que se repita: ";
	cin >> respuesta;
	for (int i=0; i<respuesta; i++) {
		cout << "Ingresar la base:";
		cin >> Base;
		if (Base > 0){
			cout << "Ingresar la altura:";
			cin >> Altura;
			if (Altura > 0) {
				Area= Altura*Base;
				cout << "Area = " << Area << "\n";
				cout << "/////////////////////////////.\n";
			} else {
				cout << "Error en la altura.\n";
				cout << "///////////////////////.\n";
			}
		} else {
			cout << "Eror en la base.\n";
			cout << "///////////////////////.\n";
		}
	}		
	
	return 0;
}

File: 1.FDP/2° Competencia/Java/Repeticion/Ejercicio2.java
import java.util.Scanner;	

public class Ejercicio2 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		for (int i=0; i<3; i++) {
			System.out.println("Calcular el area de un rectagulo.\n");
			double Base, Altura, Area;
			System.out.println("Base: ");
			Base= scan.nextDouble();
			if (Base > 0) {
				System.out.println("Altura: ");
				Altura= scan.nextDouble();
				if (Altura > 0) {
						Area= Base*Altura;
					System.out.println("Area= " + Area);
					System.out.println("////////////////////////////.");
				} else {
					System.out.println("Error en la altura");
					System.out.println("//////////////////////////");
				}
			} else {
				System.out.println("Error en la base");
				System.out.println("//////////////////////////");
			}	
		}
	}
}